OsmAnd/res/values/strings.xml

```diff
@@ -497,8 +496,8 @@
     • Other bugfixes\n\n
     </string -->
     <string name="increase_search_radius_to">Extend search radius to %1$s</string>
-    <string name="send_search_query_description"><![CDATA[Your search query will be sent to: <b>\"%1$s\"</b>, along with your location.<br/><br/>
-        Personal info is not collected, only search data needed to improve the search.<br/>]]></string>
+    <string name="send_search_query_description"><![CDATA[Your search query will be sent to: <b>\"%1$s\"</b>, along with your location.\n\n
```

Comment:
Not 100% sure here, but it is consistent.

Reply:
I think \n won't work in this case